### PR TITLE
[4.0] fix the status dropdown arrow colour

### PR DESCRIPTION
**Pull request by Harshil99**
Pull Request for Issue #23793 .

### Summary of Changes
Changed the background (which is arrows) of both success and danger custom select.

### Testing Instructions
Do these changes and then compile with npm ci.

### Expected result
Dropdown arrows have grey colour in enabled and disabled state.

### Actual result
Color changed to white.

### Documentation Changes Required
No

### CODE REVIEW

**Comment by C-Lodder** [comment]
Please don't just whack SCSS into the general `template.scss` file. Use the appropriate files. If you can't be bothered to search, just generate a source map using the build tools.

In this case, use this file: `administrator/templates/atum/scss/vendor/bootstrap/_custom-forms.scss`

**Comment by Harshil99** [comment]
Sorry for that. I will update the PR with changes.

**Review by C-Lodder — APPROVED** [review]

File: administrator/templates/atum/scss/vendor/bootstrap/_custom-forms.scss [hunk 1]
```diff
@@ -5,6 +5,7 @@
   &.custom-select-color-state {
 
     &.custom-select-success {
+        background: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 4 5'%3e%3cpath fill='%23fff' d='M2 0L0 2h4zm0 5L0 3h4z'/%3e%3c/svg%3e") no-repeat right 1rem center/8px 10px;
```
Comment (infograf768) [inline administrator/templates/atum/scss/vendor/bootstrap/_custom-forms.scss]: Please align to lines below

File: administrator/templates/atum/scss/vendor/bootstrap/_custom-forms.scss [hunk 2]
```diff
@@ -16,6 +17,7 @@
 	}
 
     &.custom-select-danger {
+    background: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 4 5'%3e%3cpath fill='%23fff' d='M2 0L0 2h4zm0 5L0 3h4z'/%3e%3c/svg%3e") no-repeat right 1rem center/8px 10px;
```
Comment (infograf768) [inline administrator/templates/atum/scss/vendor/bootstrap/_custom-forms.scss]: Please align to lines below

**Comment by infograf768** [comment]
Works fine. Will test OK when lines aligned.

**Comment by Harshil99** [comment]
@infograf768 I have aligned that line to below lines.

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 70539cf1f4542e6b9ecc22c35b15ce388a4c7c23<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24099">issues.joomla.org/tracker/joomla-cms/24099</a>.</sub>

**Comment by Harshil99** [comment]
@infograf768 Still any changes required ?

**Comment by roland-d** [comment]
@Harshil99 Can you please fix the merge conflict so I can get this tested?

**Comment by brianteeman** [comment]
@roland-d with the new template this PR is not needed and should be closed

**Pull request closed without merging**